release/README.md

```diff
@@ -0,0 +1,12 @@
+# Release Process
+KFServing's automated release processes run in Google Cloud Build under the project `kfserving`. For permissions to this project, please contact ellisbigelow@google.com.
+
+Builds are available at https://console.cloud.google.com/cloud-build/builds?project=kfserving.
+To view builds, you must be a member of kubeflow-discuss@googlegroups.com.
+
+## Latest Release
+KFServing's release process relies on Github build triggers in Google Cloud Build. This build trigger was configured manually to execute the configurations in cloud-build-configs. For each build config, whenever a change is made to this repository, a Cloud Build is triggered which rebuilds all relevant images with a tag and pushes them to `gcr.io/kfserving`. For example, `kfserving-controller.cloud-build.yaml` will result in an image called `gcr.io/kfserving/kfserving-controller:latest`
+
```

Comment:
now, all kubeflow images are released in gcr.io/kubeflow-images-public/, will kfserving be the special one?

Reply:
I'm happy to align. I'm not sure which project owns kubeflow-images-public.

Reply:
Actually, I had no idea about it, too. Maybe @jlewi should know.

Reply:
I believe until 0.1, we can keep it in gcr.io/kfserving.